python/doc/examples/reliability_sensitivity/sensitivity_sobol.ipynb

```diff
@@ -29,29 +29,29 @@
    "source": [
     "## Introduction\n",
     "\n",
-    "In this example we are going to quantify the correlation between the input variables and the output variable of a model thanks to the Sobol indices.\n",
+    "In this example we are going to quantify the correlation between the input variables and the output variable of a model thanks to Sobol indices.\n",
     "\n",
-    "The Sobol indices allow to evaluate the importance of a single variable or a specific set of variables. Here the Sobol indices are estimated by sampling, from two input samples and a numerical function.\n",
+    "Sobol indices are designed to evaluate the importance of a single variable or a specific set of variables. Here the Sobol indices are estimated by sampling from the two input samples and propagating uncertainty through a numerical function.\n",
```

Comment:
Which two input samples? Shouldn't we write something like this:
```suggestion
    "Sobol indices are designed to evaluate the importance of a single variable or a specific set of variables. Here the Sobol indices are estimated by sampling from the distributions of the input variables and propagating uncertainty through a numerical function.\n",
```

Reply:
Fixed.